Documentation/build_process.md

```diff
@@ -501,6 +501,67 @@ when packaing Release applications.
 
     Added in Xamarin.Android 7.1.
 
+-   **AndroidVersionCodePattern** &ndash; A string property which allows
+    the developer to customize the `versionCode` in the manifest.
+    See [Creating the Version Code for the APK](https://developer.xamarin.com/guides/android/advanced_topics/build-abi-specific-apks/#Creating_the_Version_Code_for_the_APK)
+    for information on deciding a `versionCode`.
+    
+    Some examples, if `abi` is `armeabi` and `versionCode` in the manifest
+    is `123`
+
+        {abi}{versionCode}
+
+    will prodice a versionCode of `1123`.
+    If `abi` is `x86_64` and `versionCode` in the manifest
+    is `44`. This will produce `544`.
+
+    If we include a left padding format string 
+
+        {abi}{versionCode:0000}
+
+    it would produde `50044` because we are left padding the `versionCode`
+    with `0`. Alternatively you can use the decimal padding such as
+
+        {abi}{versionCode:D4}
+
+    which does the same as the previous example. 
+
+    Only '0' and 'Dx' padding format strings are supported since the value
+    MUST be an integer.
+    
+    Pre defined key items
+        
+    -   **abi**  &ndash; Inserts the targetted abi for the app
+        -   1 &ndash; `armeabi`
+        -   2 &ndash; `armeabi-v7a`
+        -   3 &ndash; `x86`
+        -   4 &ndash; `arm64-v8a`
+        -   5 &ndash; `x86_64`
+
+    -   **minSDK**  &ndash; Inserts the minimum supported Sdk
+        value from the `AndroidManifest.xml` or `11` if none is
+        defined.
+
+    -   **versionCode** &ndash; Uses the version code direrctly from 
+        `Properties\AndroidManifest.xml`. 
+
+    You can define custom items using the [AndroidVersionCodeProperties](#AndroidVersionCodeProperties) 
+    property.
+
+    Added in Xamarin.Android 7.2.
+
+-   **AndroidVersionCodeProperties** &ndash; A string property which allows
+    the developer to define custom items to use with the [AndroidVersionCodePattern](#AndroidVersionCodePattern).
```

Comment:
Looks like a prior `documentation` repro which moved this to be a list instead of a table *broke the hyperlinks*, or, hyperlinks now require post-processing/hosting on developer.xamarin.com

Specifically, this link doesn't go anywhere when simply rendered as HTML.

Perhaps this should add `<a name="..." />` so that these links work?